lib/routes/web/series.ts

```diff
@@ -0,0 +1,34 @@
+import type { Data, Route } from '@/types';
+import { fetchItems, hyphen2Pascal } from './utils';
+
+export const route: Route = {
+    path: '/series/:seriesName',
```

Comment:
Missing `parameters` property to explain `seriesName`
docs: https://docs.rsshub.app/joinus/new-rss/start-code#creating-route

Reply:
thanks for your review